src/components/calculator.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export default class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { display: 0 };
+  }
+
+  render() {
+    const { display } = this.state;
+    return (
+      <div className="calc">
+        <span>{display}</span>
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
+        <button type="button">%</button>
+        <button type="button" className="colored">+</button>
+        <button type="button">7</button>
+        <button type="button">8</button>
+        <button type="button">9</button>
+        <button type="button" className="colored">x</button>
+        <button type="button">4</button>
+        <button type="button">5</button>
```

Comment:
Please ensure that the calculator is aligned to the center of the page according to the project requirement.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/158921514-95a53bbb-fbcd-4610-b204-9e1c7aa75801.png) | ![image](https://user-images.githubusercontent.com/10332499/158921563-086893b0-0d2e-45be-802a-81f023a2e245.png)

Reply:
It is centered now, but I didn't see that specified in the requirements.